### ISSUE TITLE
UI Shell is not accessible with Windows High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6897]
## Environment

> Operating system

Windows 10 latest

> Browser

Edge, Firefox

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description
The down chevron on the sidebar is inaccessible (too low contrast
On firefox (like we've seen elsewhere) there are no focus indicators

Firefox
![image](https://user-images.githubusercontent.com/40970507/93948267-9ed96d00-fd03-11ea-96d6-576e4f1fd93c.png)

Edge
![image](https://user-images.githubusercontent.com/40970507/93948272-a3058a80-fd03-11ea-9ec2-d7e532611671.png)
